datamodel/low/base/schema.go

```diff
@@ -1075,21 +1078,15 @@ func buildSchema(schemas chan schemaProxyBuildResult, labelNode, valueNode *yaml
 						return
 					}
 				}
-				refBuilds++
-				go build(vn, vn, syncChan, isRef, refLocation)
-			}
-			completedBuilds := 0
-			for completedBuilds < refBuilds {
-				select {
-				case res := <-syncChan:
-					completedBuilds++
-					schemas <- schemaProxyBuildResult{
-						k: low.KeyReference[string]{
-							KeyNode: labelNode,
-							Value:   labelNode.Value,
-						},
-						v: *res,
-					}
+				// We need to build arrays sequentially so call it in the loop instead of goroutines
```

Comment:
Hi @daveshanley . Sorry, i was on vacation.I tried doing benchmark, but on both (stripe and k8s) specs lib finds circular refs. (I followed `BenchmarkNewDocument` example for benchmarking)

I assume I need to wait for #64 to be merged. Or there is another approach: fork k8s spec and fix circular there then base test on it. What do you prefer ?

Reply:
(Just jumping in to note that the Stripe API spec that's in the repo does actually contain some *invalid* infinite circular references, so that will still fail to load until I get around to adding the options to treat these as warnings rather than errors. The k8s spec should work correctly once our PR is completed.)

Reply:
@BenjaminNolan Do you have any ETA on PR being merged?

Reply:
Hey just want to jump in here, I have been working on an alternate PR that retains the async building nature but still retains the original order so shouldn't have an impact on performance. I am happy to continue with that and get that PR up ASAP which should reduce the need for benchmarking as we won't be changing the performance characteristics

Reply:
PR here https://github.com/pb33f/libopenapi/pull/66

Reply:
@TristanSpeakEasy  Fantastic. When can you merge #66?